src/formatDate.js

```diff
@@ -50,7 +50,40 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const formattedArray = [];
+
+  const dateInput = {
+
+    dataConverter(format, value) {
```

Comment:
The 'dataConverter' function is part of the 'dateInput' object. Consider using a more descriptive name, such as 'convertDateComponent'.